### DISCUSSION TITLE
How to convert .NWB to .mda

### DISCUSSION THREAD

**Discussion by wentao-uw** [NeurodataWithoutBorders/helpdesk#44] [Q&A] [answered]
Hi, 
  I am trying to use mountainsort4, but the data format is .mda and I only have .NWB file. How to convert .NWB to .mda?

**Comment by oruebel** [comment, marked as answer]
I believe SpikeInterface https://spikeinterface.readthedocs.io/ support mountainsort4 so that you should be able to use the .NWB directly without having to convert to .mda first
